pegasus/sites.v3/code.org/public/teach.haml

```diff
@@ -1 +1,181 @@
-- redirect CDO.studio_url('/courses?view=teacher', CDO.default_scheme)
+---
```

Comment:
This is just the content in `teach-draft.haml` copied over to this file!

Reply:
Thanks for this!